spec/models/trainee_spec.rb

```diff
@@ -150,4 +150,31 @@
       it_behaves_like "a working search"
     end
   end
+
+  describe "#ordered_by_date" do
+    let(:trainee_one) { create(:trainee, updated_at: 1.hour.ago) }
+    let(:trainee_two) { create(:trainee, updated_at: 1.day.ago) }
+
+    it "orders the trainess by updated_at in descending order" do
+      expect(Trainee.ordered_by_date).to match_array([trainee_two, trainee_one])
```

Comment:
What happens if you change this test to:

```ruby
expect(Trainee.ordered_by_date).to match_array([trainee_one, trainee_two])
```

I have a feeling that this will still pass - i think `match_array` doesn't care about ordering.

You might need `to eq()` , which should check order.

Reply:
Ahh good catch!

Reply:
Updated